source/MaterialXView/Main.cpp

```diff
@@ -273,16 +285,9 @@ int main(int argc, char* const argv[])
         viewer->setBakeHeight(bakeHeight);
         viewer->setBakeFilename(bakeFilename);
         viewer->setSRGBBuffer(srgbBuffer);
-        viewer->initialize();
-        if (!bakeFilename.empty()) 
-        {
-            viewer->bakeTextures();
-            viewer->requestExit();
-        } 
-        else 
-        {            
-            viewer->setVisible(true);
-        }
```

Comment:
Hi @bernardkwok Looks like this change was inadvertently deleted as a result the viewer does not initialize.